client/package.json

```diff
@@ -108,7 +108,7 @@
     "start-production": "NODE_ENV=production node server.js",
     "test": "jest",
     "coveralls": "cat coverage/lcov.info | coveralls",
-    "lint": "eslint app server.js && stylelint src && sass-lint -v",
+    "lint": "eslint app server.js && stylelint --allow-empty-input src && sass-lint -v",
```

Comment:
@foot @fbarl Please see if we want to change in the future. I mostly just wanted to help us fix some of the security issues before we get out another release. (10.0.0 of stylelint errored out on empty input, it looks like we were not feeding it the right list of files for some time?)

Reply:
ccing @bboreham too.